### PR TITLE
Update to Pulsar 2.7.2

### PR DESCRIPTION
**Pull request by eolivelli**
Update the Helm Chart according to the release process
https://github.com/apache/pulsar/wiki/Release-process#11-release-helm-chart

### CODE REVIEW

File: dev-func-k8s-values.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+enableAntiAffinity: no
+enableTls: no 
+enableTokenAuth: no
+restartOnConfigMapChange:
+  enabled: yes
+extra:
+  function: yes
+  burnell: yes
+  burnellLogCollector: yes
```
Comment (lhotari) [inline dev-func-k8s-values.yaml:26]: Remove?

File: dev-func-k8s-values.yaml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+pulsarAdminConsole:
+  replicaCount: 1
+
+kube-prometheus-stack:
+  enabled: yes
+  prometheusOperator:
+    enabled: true
+  grafana:
+    enabled: true
+    adminPassword: e9JYtk83*4#PM8
```
Comment (lhotari) [inline dev-func-k8s-values.yaml:109]: password intentional?

File: examples/test-kube-functions-minikube.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    tag: 2.8.0-SNAPSHOT
+  broker:
+    repository: apachepulsar/pulsar-all
+    tag: 2.8.0-SNAPSHOT
+  functions:
+    repository: apachepulsar/pulsar-all
+    tag: 2.8.0-SNAPSHOT
+  proxy:
+    repository: apachepulsar/pulsar-all
+    tag: 2.8.0-SNAPSHOT
```
Comment (lhotari) [inline examples/test-kube-functions-minikube.yaml:78]: Is this file intentionally part of this PR?

File: dev-func-k8s-values.yaml (hunk 1 above)
Comment (eolivelli) [reply dev-func-k8s-values.yaml]: I committed unintentionally other files on my work directory.
Removed

File: examples/test-kube-functions-minikube.yaml (hunk 3 above)
Comment (eolivelli) [reply examples/test-kube-functions-minikube.yaml]: I committed unintentionally other files on my work directory.
Removed

File: dev-func-k8s-values.yaml (hunk 2 above)
Comment (eolivelli) [reply dev-func-k8s-values.yaml]: I committed unintentionally other files on my work directory.
Removed

**Comment by eolivelli** [comment]
@lhotari PTAL again please

**Review by lhotari — APPROVED** [review]

**Pull request merged**